scripts/eqtl_hail_batch/launch_eqtl_spearman.py

```diff
@@ -0,0 +1,1501 @@
+#!/usr/bin/env python3
+# flake8: noqa: E402
+"""
+Create a Hail Batch workflow for all the EQTL analysis, including:
+
+- generate_eqtl_spearman
+- conditional_analysis rounds
+
+For example:
+
+    python3 scripts/hail_batch/eqtl_hail_batch/launch_eqtl_spearman.py \
+        --input-files-prefix 'gs://cpg-tob-wgs-test/scrna_seq/grch38_association_files' \
+        --chromosomes '1 2' \
+        --genes B_intermediate
+"""
+import os
+import logging
+
+
+def setup_logger(name):
+    loggers_to_silence = [
```

Comment:
@lgruen, this is the only way I could figure out to properly silence these. Just the `getLogger` didn't work for me.